docs/subnet/resources/repos.md

```
@@ -0,0 +1,17 @@
+---
+title: Repos
+---
+
+## Subnet Repositories
+
+Subnet Generator/Deployment Wizard: https://github.com/XinFinOrg/Subnet-Deployment
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)